src/fatx_internal.h

```diff
@@ -75,7 +81,18 @@
 #define MAX(a,b) ( ( (a) >= (b) ) ? (a) : (b) )
 
 /*
- * The superblock, as it appears on disk.
+ * The refurb info as it appears on disk.
+ */
+#pragma pack(1)
+struct fatx_refurb_info {
+    uint32_t signature;
+    uint32_t number_of_boots;
+    uint64_t first_power_on;
```

Comment:
`first_power_on` isn't set by the system, it's set during manufacturing as a format date.

Reply:
Tbh I don't have a untouched box from the factory to counter this (nor do I think it matters) -- I just picked the name based on the fact that the dash initializes the timestamp if it isn't already set:

![image](https://user-images.githubusercontent.com/9522648/154183261-0fafa438-73d9-4682-b033-334222ba8463.png)

This is `xboxdash.xbe` I pulled off my retail and applied FLIRT + some additional cleanup to ^

Reply:
til